backend/src/v5/services/mailer/templateNames.js

```diff
@@ -15,8 +15,10 @@
  *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
 
-const { v4Path } = require('../../interop');
-// eslint-disable-next-line import/no-dynamic-require, security/detect-non-literal-require, require-sort/require-sort
-const Mailer = require(`${v4Path}/mailer/mailer`);
+const TemplateNames = {};
```

Comment:
just for the sake of pointing out obvious, i don't mean implementating  the html/subject functions here, just import them into this file instead (and can do it up top)

Reply:
I now understand what you mean. But in that case why can I not get rid of the name and pass the whole template in sendEmail method ? 

![image](https://user-images.githubusercontent.com/71635617/162440675-a0cf0805-2123-4a5d-9d11-16201737f5c1.png)

then I can use template.html and template.subject directly in sendEmail without having to even get it from constants

Reply:
Because you don't want to just take the word of the callee for it - if we do that we're just calling whatever function that is passed down to the function - which could be naything

We want to check it's a valid template.

Reply:
I see...